packages/@dcl/inspector/src/components/EntityInspector/SceneInspector/Layout/Layout.css

```diff
@@ -0,0 +1,67 @@
+.SceneLayout .display {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin-bottom: 20px;
+}
+
+.SceneLayout .display h2 {
+  margin-bottom: 10px;
+}
+
+.SceneLayout .display .error {
+  color: var(--primary-main);
+  margin: 3px 0;
+  font-size: 12px;
+}
+
+.SceneLayout .display span,
+.SceneLayout .Block .limitations span {
+  text-align: center;
+  color: var(--base-10);
+  margin-bottom: 10px;
+}
+
+.SceneLayout .Grid {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
```

Comment:
Just a heads up, when the amount of parcels is wider than the width of the Grid (which is the width of the sidebar) some parecels are not reachable, even by using the scroll.

This seems like an edge case tho, and the sidebar can be widened to make them fit, just wanted to give a heads up.

https://github.com/decentraland/js-sdk-toolchain/assets/2781777/5a5858ed-228e-4ace-bb84-69364ad7b948